src/style.scss

```
@@ -107,55 +112,24 @@ a,
   }
 }
 
+// TODO: Move this and ::placeholder to a new BaseInput component
+.input {
+  @apply outline-none text-skin-link bg-transparent border-none;
```

`outline-none` is redundant because it's set globally [here](https://github.com/snapshot-labs/snapshot/pull/1777/files#diff-c0a5ddd5b365322730802e8cb15c95c3b6e965cbd3e569c69a2f089753844cdbR45-R47). But I might be wrong... input's have always and probably will forever behave.. strange. :D

```suggestion
  @apply text-skin-link bg-transparent border-none;
```